spec/spec_helper.rb

```diff
@@ -37,13 +37,13 @@ def create_claim_with_url name,url, subcards={}
 def create_claim name, subcards={}
   Card::Auth.as_bot do
     sourcepage = create_page 'http://www.google.com/?q=wikirateissocoolandawesomeyouknow'
-    Card.create! :type_id=>Card::ClaimID, :name=>name, 
-                 :subcards=>{ '+source' => {:content=>"[[#{sourcepage.name}]]",:type_id=>Card::PointerID}}.merge(subcards)    
+    Card.create! :type_id=>Card::ClaimID, :name=>name,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.